share/revng/test/tests/analysis/ImportFromCAnalysis/struct-with-a-broken-annotation/expected-error.txt

```diff
@@ -0,0 +1,6 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import-from-c: Ignoring non-integer value of an integer annotation: 'sixteen'.
```

Comment:
`of an integer _START_AT annotation`

Reply:
Sadly, this is not possible. The code for parsing these is shared, and it's definitely not easy (if even possible) to extract macro names like `_START_AT` from clang. Annotations themselves (which macros unwrap into) is a lot easier, so I used this one for now, even though I'm not sure how useful those would be for the user.

Eventually, we might want to maintain are mapping between the two for the users' sake, but even the thought makes me kind of sad.

Reply:
Ah damn, macros!!!

I understand that they are a pain, but at the moment, if I put myself in a user's shoes, what I see it the following situation:
- in C for specifying types I'm instructed to use macros such as `_START_AT` and others (like `_REG`, `_STACK`)
- in most error messages I can just see the macros directly (again `_STACK`, `_REG`)
- I never see the `field_start_offset` anywhere in the C code generated by `revng` and I don't know how to use it
- I get error messages containing `field_start_offset` when parsing my C code, whereas I never used `field_start_offset` at all

This is very confusing.

So, I appreciate the effort to make the code generic, but this currently degrades the UX.

The mapping for the users' sake at this point is important, and it's very easy to implement. So please do it.
You don't need to care about making it extensible by user-defined annotations yet. Just go with a simple design. Even a `static const` global map is fine.

Reply:
Closing this as done. Thank you.